explorer/lib/explorer_web/live/pages/batches/index.ex

```diff
@@ -17,18 +17,37 @@ defmodule ExplorerWeb.Batches.Index do
      assign(socket,
        current_page: current_page,
        batches: batches,
+       next_batch_countdown_progress: Helpers.next_block_progress(),
+       time_to_next_block: Helpers.time_to_next_block(),
        last_page: Batches.get_last_page(@page_size),
        page_title: "Batches"
      )}
   end
 
+  # 12.22222 316
```

Comment:
?

Reply:
Lol, that was a reminder for myself when testing, didn't notice it was still there. I'll delete it.